helm/prometheus-rules/templates/alerting-rules/kube-state-metrics.rules.yml

```diff
@@ -28,6 +56,27 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    - alert: KubeStateMetricsNotRetrievingMetrics
+      annotations:
+        description: '{{`KubeStateMetrics ({{ $labels.instance }}) is not retrieving metrics.`}}'
+        opsrecipe: kube-state-metrics-down/
+      expr: |-
+        # When it looks up but we don't have metrics
+        count({app="kube-state-metrics"}) < 10
+      for: 60m
+      labels:
+        area: kaas
+        cancel_if_apiserver_down: "true"
+        cancel_if_cluster_status_creating: "true"
+        cancel_if_cluster_status_deleting: "true"
+        cancel_if_cluster_has_no_workers: "true"
+        inhibit_kube_state_metrics_down: "true"
+        cancel_if_kubelet_down: "true"
+        cancel_if_kube_state_metrics_down: "true"
+        cancel_if_outside_working_hours: "false"
+        severity: page
+        team: atlas
+        topic: observability
```

Comment:
When this condition was added, we wanted it to inhibit KSM-dependent alerts.
But I think, if it takes 60min to fire, it won't prevent KSM-dependent alerts from firing.

If we're afraid it would be too noisy, we can try setting it to not page outside of business hours. WDYT?

Reply:
I guess we can both set it not to page outside of business hours and reduce its time to page to 30min

Reply:
It's been changed to:
- 20min
- cancel_if_outside_working_hours

Fine with me :smiley: